contrib_bots/lib/github_issues.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import absolute_import
+import requests
+import json
+from future import standard_library
+standard_library.install_aliases()
+import urllib.request, urllib.error, urllib.parse
+import six.moves.configparser
+import os
+from . import github
+
+class IssueHandler(object):
+    '''
+    This plugin facilitates sending issues to github, when
+    an item is prefixed with '@issue' or '@bug'
+
+    It will also write items to the issues stream, as well
+    as reporting it to github
+    '''
+
+    url = 'https://api.github.com/repos/{}/{}/issues'
```

Comment:
By convention Python constants should be in capital letters and words should be separated by underscores:
https://www.python.org/dev/peps/pep-0008/#constants